web/src/components/profile/ProfileAddress.test.tsx

```diff
@@ -0,0 +1,824 @@
+            formationFormData: generateFormationFormData({ ...address }),
+          }),
+        })
+      );
+      renderNjBusinessComponent({});
+
+      expect(screen.getByLabelText("Address line1")).toHaveValue("");
+      fillText("Address line1", "1111 Home Alone");
+      expect(screen.getByLabelText("Address line1")).toHaveValue("1111 Home Alone");
+    });
+
+    it("renders profile address line 1 error", () => {
+      const address = generateAddress({
+        addressLine1: "",
+      });
+      useMockBusiness(
+        generateBusiness({
+          formationData: generateFormationData({
+            completedFilingPayment: false,
+            formationFormData: generateFormationFormData({ ...address }),
+          }),
+        })
+      );
+      renderNjBusinessComponent({});
+
+      blurInput("Address line1");
+      expect(screen.getByText(Config.formation.fields.addressLine1.error)).toBeInTheDocument();
+    });
+
+    it("renders profile address line 2 field with input value", () => {
+      const address = generateAddress({
+        addressLine2: "1111 Home Alone",
+      });
+      useMockBusiness(
+        generateBusiness({
+          formationData: generateFormationData({
+            completedFilingPayment: false,
+            formationFormData: generateFormationFormData({ ...address }),
+          }),
+        })
+      );
+      renderNjBusinessComponent({});
+
+      expect(screen.getByLabelText("Address line2")).toHaveValue("1111 Home Alone");
+    });
+
+    it("renders profile address line 2 reflects changed value", () => {
+      const address = generateAddress({
+        addressLine2: "",
+      });
+      useMockBusiness(
+        generateBusiness({
+          formationData: generateFormationData({
+            completedFilingPayment: false,
+            formationFormData: generateFormationFormData({ ...address }),
+          }),
+        })
+      );
+      renderNjBusinessComponent({});
+
+      expect(screen.getByLabelText("Address line2")).toHaveValue("");
+      fillText("Address line2", "1111 Home Alone");
+      expect(screen.getByLabelText("Address line2")).toHaveValue("1111 Home Alone");
+    });
+
+    it("renders profile address line 2 error", () => {
+      const address = generateAddress({
+        ...emptyAddressData,
+        addressLine2: "123",
+      });
+      useMockBusiness(
+        generateBusiness({
+          formationData: generateFormationData({
+            completedFilingPayment: false,
+            formationFormData: generateFormationFormData({ ...address }),
+          }),
+        })
+      );
+      renderNjBusinessComponent({});
+
+      blurInput("Address line2");
+      expect(screen.getByText(Config.formation.fields.addressLine1.error)).toBeInTheDocument();
+      expect(screen.getByText(Config.formation.fields.addressMunicipality.error)).toBeInTheDocument();
+      expect(screen.getByText(Config.formation.fields.addressZipCode.error)).toBeInTheDocument();
```

Comment:
[pebble]: I was anticipating an expect statement for `addressLine2.error` in here. That probably needs to be the only expect statement here or the test name should probably be changed to reflect these 3 expects.